### PR TITLE
Added missing fields into activity.Info

### PR DESCRIPTION
**Pull request by mfateev**

### CODE REVIEW

File: internal/internal_activity.go [hunk 1]
```diff
@@ -90,14 +91,19 @@ type (
...
-		isLocalActivity   bool
+		taskToken          []byte
+		workflowExecution  WorkflowExecution
+		activityID         string
+		activityType       ActivityType
+		serviceInvoker     ServiceInvoker
+		logger             *zap.Logger
+		metricsScope       tally.Scope
+		isLocalActivity    bool
+		heartbeatTimeout   time.Duration
```
Comment (madhuravi) [inline internal/internal_activity.go]: How about embedding activityInfo in here instead of duplicating the fields?

File: testsuite/activity.go [hunk 2]
```diff
@@ -37,9 +37,10 @@ type ServiceInvoker = internal.ServiceInvoker
 func WithActivityTask(
```
Comment (yiminc) [inline testsuite/activity.go]: this function is not used anywhere, we should remove it.

File: internal/internal_public.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ type (
 		// - RespondActivityTaskCompletedRequest
 		// - RespondActivityTaskFailedRequest
 		// - RespondActivityTaskCancelRequest
-		Execute(task *s.PollForActivityTaskResponse) (interface{}, error)
+		Execute(taskList string, task *s.PollForActivityTaskResponse) (interface{}, error)
```
Comment (madhuravi) [inline internal/internal_public.go]: We can make tasklist also part of PollForActivityTaskResponse. All other params are in there, tasklist seems relevant as well.

Comment (mfateev) [reply internal/internal_public.go]: I think it is overkill as poller knows which task list it polls on.

File: internal/internal_activity.go (hunk 1 above)
Comment (mfateev) [reply internal/internal_activity.go]: I think it is mostly about layering. ActivityInfo doesn't exist at the level of this function.

Comment (madhuravi) [reply internal/internal_activity.go]: Not sure what you mean? They're both in the same package, struct definitions can be embedded.

File: internal/internal_public.go (hunk 3 above)
Comment (madhuravi) [reply internal/internal_public.go]: Makes sense.

**Pull request merged**